### ISSUE TITLE
This bit is hard to read

### ISSUE THREAD

**Issue by RichardLitt** [licensezero/parity-public-license#1]
> To release source code, promptly ensure the preferred form for
making changes is published through a widely used, freely
accessible source code distribution system under a license at
least as permissive as this license.

Specifically,

> ... promptly ensure the preferred form for
making changes is published through ...

Caught me up a few times before I got it. Maybe reword to make reading easier?

**Comment by kemitchell** [comment]
Thank you for this feedback!

I shouldn't be surprised to hear this part is rough. I think the trouble started when I rewrote with "ensure" to make clear that if something's already been released, you don't have to release it again.

I'll have a proposed fix for you in a jiffy.

PS: Would you be alright with me adding you to a `THANKS` file here in the repo?

**Comment by RichardLitt** [comment]
I don't mind at all. :)
